sdk/java-sdk-spring/src/it/java/com/example/wiring/pubsub/DockerIntegrationTest.java

```diff
@@ -52,18 +50,27 @@ public abstract class DockerIntegrationTest {
 
   private KalixSpringApplication kalixSpringApplication;
 
-  public DockerIntegrationTest(ApplicationContext applicationContext) {
+
```

Comment:
This changes are related to the `TracingIntegrationTest`. I needed to pass some extra configuration.